co-authors-plus.php

```diff
@@ -1644,8 +1644,28 @@ public function filter_author_archive_title( $title ) {
 		$author = $this->get_coauthor_by( 'user_nicename', $author_slug );
 		
 		return sprintf( __( 'Author: %s' ), $author->display_name );
+	}	
+
+	/**
+	 * Get the post count for the guest author
+	 *
+	 * @global oebject $coauthors_plus
```

Comment:
I don't think this phpdoc comment is necessary anymore since we've moved the function into the `CoAuthors_Plus` class.